src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/NuGet.SolutionRestoreManager.Interop.csproj

```diff
@@ -3,23 +3,28 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
 
   <PropertyGroup>
+    <PackageId>NuGet.SolutionRestoreManager.Interop</PackageId>
+    <PackageVersion>$(Version)</PackageVersion>
     <TargetFramework>net46</TargetFramework>
     <Shipping>true</Shipping>
     <PackProject>true</PackProject>
     <IncludeInVsix>true</IncludeInVsix>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
```

Comment:
this is not supported, remove

Reply:
In the package generated nuspec I see the owners metadata. 

Is this a bug or something funny going on?

Reply:
owners defaults to whatever the value of authors is but currently there is no way to set it from a sdk project.

Reply:
Ah, makes sense. Will remove.